content/blog/2020-08-10/august-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,205 @@
+
+### DVC 1.0 migration script
+
+Our friends at [DAGsHub](https://dagshub.com/) have released a script to help
+DVC users upgrade their pipelines to the new DVC 1.0 format! Says Simon, a
+DAGsHub engineer, in his tutorial:
+
+> In this post, I'll walk you through the process of migrating your existing
+> project from DVC ≤ 0.94 to DVC 1.X using a single automated script, and then
+> demonstrate a way to check that your migration was successful.
+
+Read the blog and get migrating (but don't worry if you can't; DVC 1.0 is
+backwards compatible). <external-link
+href="https://towardsdatascience.com/automatically-migrate-your-project-from-dvc-0-94-to-dvc-1-x-416a5b9e837b"
+title="Automatically migrate your project from DVC≤ 0.94 to DVC 1.x"
+description="Migrating your project from DVC ≤ 0.94 to DVC 1.x can be a very involved process. Here’s an easy way to do it."
+link="medium.com"
+image="/uploads/images/2020-08-10/dagshub.jpg"/>
+
+### Recommended reading
+
+Here are some of our favorite blogs from around the internet 🌏.
+
+- [Déborah Mesquita](https://www.deborahmesquita.com/), data scientist (and an
+  excellent writer to follow), published a tutorial about DVC pipelines that is
+  truly deserving of the moniker "ultimate guide". It's a start-to-finish case
+  study about a typical machine learning project, with DVC pipelines to automate
+  everything from grabbing the data to training and evaluating a model. Also, it
+  comes with a video tutorial if you prefer to watch instead of read!
+
+<external-link
+href="https://towardsdatascience.com/the-ultimate-guide-to-building-maintainable-machine-learning-pipelines-using-dvc-a976907b2a1b"
+title="The ultimate guide to building maintainable Machine Learning pipelines using DVC"
+description="Learn the principles for building maintainable Machine Learning pipelines using DVC"
+link="medium.com"
+image="/uploads/images/2020-08-10/deborah.jpg"/>
+
+- Software engineer
+  [Vaithy Narayanan](https://www.linkedin.com/in/vaithyanathan/) created the
+  first ever ☝️ CML user blog! Vaithy created a pipeline that covers data
+  collection to model training and testing, and used CML to automate the
+  pipeline execution whenever the project's GitHub repository is updated. He
+  ends with some insightful discussion about the strengths and weaknesses of the
+  approach.
+
+<external-link
+href="https://medium.com/@karthik.vaithyanathan/using-continuous-machine-learning-to-run-your-ml-pipeline-eeeeacad69a3"
+title="Using Continuous Machine Learning to Run Your ML Pipeline"
+description="Vaithy Narayanan"
+link="medium.com"
+image="/uploads/images/2020-08-10/vaithy.jpg"/>
+
+- [Ryan Gross](https://www.linkedin.com/in/ryan-w-gross/), a VP at Pariveda
+  Solutions, blogged about the future of data governance and the lessons from
+  DevOps that might save the day. Honestly, you should probably start reading
+  for this cover image alone. ![](/uploads/images/2020-08-10/dataops.png)
```

Comment:
separate line? 

also, please optimize the image if possible

Reply:
shrunk it in revisions w/tinypng